### PR TITLE
- avoiding JNDI errors when testing with uberfire code

### PR DESCRIPTION
**Pull request by mrietveld**
[This use](https://github.com/uberfire/uberfire/blob/master/uberfire-commons/src/main/java/org/uberfire/commons/async/SimpleAsyncExecutorService.java#L75) of the `InitialContext` is causing errors in my tests, and `simple-jndi` (scope:test) fixes them. :) 

See for more info: http://fandry.blogspot.nl/2011/03/junit-based-integration-testing-with.html

### CODE REVIEW

File: kie-third-party-bom/pom.xml [hunk 1]
```diff
@@ -742,6 +742,18 @@
         <version>${version.com.github.detro}</version>
       </dependency>
 
+      <dependency>
+        <groupId>com.github.detro</groupId>
+        <artifactId>phantomjsdriver</artifactId>
+        <version>${version.com.github.detro}</version>
+      </dependency>
```
Comment (psiroky) [inline kie-third-party-bom/pom.xml]: This seems to be a duplicated dependency? (see the one above)

File: kie-third-party-bom/pom.xml [hunk 2]
```diff
@@ -742,6 +742,18 @@
...
+
+      <dependency>
+        <groupId>simple-jndi</groupId>
+        <artifactId>simple-jndi</artifactId>
+        <version>0.11.4.1</version>
```
Comment (psiroky) [inline kie-third-party-bom/pom.xml]: I guess this got forgotten -- it should use the version property.

File: kie-third-party-bom/pom.xml (hunk 1 above)
Comment (mrietveld) [inline kie-third-party-bom/pom.xml]: Thanks, will remove these.

**Comment by mrietveld** [comment]
@psiroky Apologies -- the PR has been fixed.

**Comment by psiroky** [comment]
+1

**Pull request merged**